cv32e40p/sim/core/Makefile

```diff
@@ -121,10 +121,10 @@ VSIM_SCRIPT_DIR	  = ../tools/vsim
 VSIM_SCRIPT       = $(VSIM_SCRIPT_DIR)/vsim.tcl
 
 # vcs configuration (hopelessly out of date)
-VCS_VERSION             = O-2018.09-SP1-1
-VCS_HOME                = /opt/synopsys/vcs-mx/$(VCS_VERSION)
+VCS_VERSION             = Q-2020.03-SP2
```

Comment:
I know they did not before this pull-request, but these variables should use conditional assignment:
```
VCS_VERSION ?= Q-2020.03-SP2
VCS_HOME      ?= /eda/synopsys/vcs/$(VCS_VERSION)
```
This will allow teams with differing installations to use this Makefile without modification (the variables can be set as shell variables).